mixins/demo/localize-mixin-mission.js

```diff
@@ -0,0 +1,24 @@
+import { html, LitElement } from 'lit';
+import { LocalizeDynamicMixin } from '../../mixins/localize-dynamic-mixin.js';
+
+class Mission extends LocalizeDynamicMixin(LitElement) {
+
+	static get localizeConfig() {
+		const langResources = {
+			'en': { mission: 'Transforming the way [a]the world[/a] learns' },
+			'fr': { mission: 'Transformer la façon dont [a]le monde[/a] apprend' }
+		};
+		return {
+			importFunc: async lang => langResources[lang]
+		};
+	}
+
+	render() {
+		const _link = 'href="https://wikipedia.org/wiki/Earth" target="_blank"';
+		return html`
+			<p>${this.localizeHTML('mission', { _link })}</p>
```

Comment:
Not getting a link from this - should it be:

```suggestion
			<p>${this.localizeHTML('mission', { link: _link })}</p>
```